NostalgiaForInfinityNext.py

```diff
@@ -2423,14 +2423,41 @@ def load_hold_trades_config(self):
                 log.error("Failed to load JSON from %s: %s", hold_trades_config_file, exc)
             else:
                 trade_ids = hold_trades_config.get("trade_ids")
+                trade_pairs = hold_trades_config.get("trade_pairs")
 
             if not trade_ids:
                 return
 
             open_trades = {
                 trade.id: trade for trade in Trade.get_trades_proxy(is_open=True)
             }
-
+            if isinstance(trade_pairs, dict):
+                # New syntax
+                for trade_pair, profit_ratio in trade_pairs.items():
+                    try:
+                        trade_pair = str(trade_pair)
+                    except TypeError:
+                        log.error(
+                            "The trade_id(%s) defined under 'trade_pairs' in %s is not a string",
+                            trade_pair, hold_trades_config_file
+                        )
+                        continue
+                    if not isinstance(profit_ratio, float):
+                        log.error(
+                            "The 'profit_ratio' config value(%s) for trade_pairs %s in %s is not a float",
+                            profit_ratio,
+                            trade_pair,
+                            hold_trades_config_file
+                        )
+                    else:
+                        self.hold_trade_ids[trade_pair] = profit_ratio
```

Comment:
Might adding `self.hold_trade_pairs` make sense?
I'll check based on you logic on the confirm sell function.

Reply:
Ok, it works, let's reuse the same variable...

Reply:
why wouldn't it works?...

Reply:
It would depend on the logic you implemented, which is good.